src/main/java/com/extrahardmode/ExtraHardMode.java

```diff
@@ -98,16 +97,40 @@ public void onEnable()
         registerModule(AntiFarming.class, new AntiFarming(this));
         registerModule(AnimalCrowdControl.class, new AnimalCrowdControl(this));
         registerModule(AntiGrinder.class, new AntiGrinder(this));
-        registerModule(DebugMode.class, new DebugMode(this));
+        
+
+        boolean ddb = false;
```

Comment:
1) variable names basically suck
2) Instead of copy-pasting code you could add a new method, and hand in the argument of the feature of your choice to check. It'd actually make this a bazillion times cleaner (just a one-line change for each module, rather than 8)
3) What's with all the extra newlines?